deployment/ansible/group_vars/all

```diff
@@ -44,7 +44,7 @@ docker_compose_version: "1.26.*"
 geop_host: "localhost"
 geop_port: 8090
 
-geop_version: "5.2.0"
+geop_version: "5.3.0"
```

Comment:
This may cause build failures until 5.3.0 is tagged and released for mmw-geoprocessing.